nxdrive/utils.py

```diff
@@ -1286,3 +1288,33 @@ def compute_digest(path: Path, digest_func: str, callback: Callable = None) -> s
         return UNACCESSIBLE_HASH
 
     return str(h.hexdigest())
+
+
+def save_config(conf_name: str, env: str, config_dump: Dict[str, Any]) -> Path:
+    """Update the configuration file with passed config dump."""
+    #  Check if config file already exist, if not then use executable parent folder
+    paths = (
+        Path(sys.executable).parent / conf_name,
+        Path(Options.nxdrive_home) / conf_name,
+        Path(conf_name),
+    )
+    res = [conf_file for conf_file in paths if conf_file.is_file()]
+    conf_path = res[0] if res else paths[0]
```

Comment:
```suggestion
    conf_path = res[0] if res else paths[1]
```
We want the config from `$HOME/.nuxeo-drive/config.ini` by default.